saltgui/static/scripts/routes/home.js

```diff
@@ -181,8 +260,26 @@ class HomeRoute extends Route {
 
     var pre = this._createDiv("unaccepted", "unaccepted");
     pre.id = "status";
-
     element.appendChild(pre);
+
+    var rejectButton = this._createDiv("run-command-button", "Reject &#9658;");
+    rejectButton.addEventListener('click', evt => {
+      this._runRejectKey(hostname, evt);
+    });
+    element.appendChild(rejectButton);
+
+    var deleteButton = this._createDiv("run-command-button", "Delete &#9658;");
+    deleteButton.addEventListener('click', evt => {
+      this._runDeleteKey(hostname, evt);
+    });
+    element.appendChild(deleteButton);
+
+    var acceptButton = this._createDiv("run-command-button", "Accept &#9658;");
+    acceptButton.addEventListener('click', evt => {
+      this._runAcceptKey(hostname, evt);
+    });
+    element.appendChild(acceptButton);
+
```

Comment:
Also here I see a lot code duplication of creating the buttons here. I think you should create some helper functions for this.

Reply:
Agreed, working on it...

Reply:
Simplified this code.